nursery/packaged-as-single-file-dotnet-application.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: packaged as single-file .NET application
+    namespace: executable/installer/dotnet
+    authors:
+      - michael.hunhoff@mandiant.com
+    description: https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli
+    scope: file
+  features:
+    - and:
+      - string: "Failure processing application bundle; possible file corruption."
+      - string: "Failure processing application bundle."
+      - string: "Bundle header version compatibility check failed"
+      - string: "A fatal error was encountered. Could not extract contents of the bundle"
```

Comment:
did you have a chance to look at VT for these strings? if not, i can do it tomorrow and try to find a good example.

Reply:
see https://github.com/mandiant/capa-testfiles/pull/164